docs/code-quality/ca1721.md

```diff
@@ -29,7 +29,7 @@ ms.workload:
 
 ## Cause
 
-The name of a member starts with 'Get' and otherwise matches the name of a property. For example, a type that contains a method that is named 'GetColor' and a property that is named 'Color' cause a rule violation.
+The name of a member starts with 'Get' and otherwise matches the name of a property and none of the 2 members are marked as `[Obsolete]`. For example, a type that contains a method that is named 'GetColor' and a property that is named 'Color' cause a rule violation.
```

Comment:
I would just add new sentence at the end of existing one `This rule will not fire if either the property or the method is marked as [Obsolete].`